src/main/java/bio/terra/service/snapshotbuilder/query/FilterVariable.java

```diff
@@ -1,3 +1,7 @@
 package bio.terra.service.snapshotbuilder.query;
 
-public interface FilterVariable extends SqlExpression {}
+import java.util.List;
+
+public interface FilterVariable extends SqlExpression {
+  List<TableVariable> getTables();
```

Comment:
why do we need to add getTables? Is it used anywhere besides in tests?

Reply:
We need it to add the tables to the query afterwards, since the query needs to contain a complete list of tables, but we won't know what that list is until after we build the query.